src/accounts/views.py

```diff
@@ -57,6 +60,17 @@ def create_profile(user=None, **kwargs):
                                                     xii_percentage=kwargs['xii_percentage'], )
         userprofile.save()
 
+        # Sends email to the registered student
+        subject = "Registered with SPC"
+        from_email = settings.EMAIL_HOST_USER
+        to_email = [user.email, ]
+        with open(settings.BASE_DIR + "/templates/accounts/student_register_email.txt") as f:
+            register_message = f.read()
+        message = EmailMultiAlternatives(subject=subject, body=register_message, from_email=from_email, to=to_email)
+        html_template = render_to_string("accounts/student_register_email.html", {'username': user.username, 'email': user.email})
```

Comment:
Doesn't the email template already have the message content or the `body` which you added above